jobs/JGLOBAL_ARCHIVE

```diff
@@ -50,6 +50,10 @@ for grid in "0p25" "0p50" "1p00"; do
        "COMIN_ATMOS_GRIB_${grid}:COM_ATMOS_GRIB_GRID_TMPL"
 done
 
+for grid in '0p25' '0p50' '1p00'; do
+  prod_dir="COMOUT_ATMOS_ENSSTAT_${grid}"
```

Comment:
Should this be a `COMIN`?

Reply:
Instead of defining `COMIN_ATMOS_ENSSTAT_${grid}` in `JGLOBAL_ARCHIVE`, is there a way to use `COM_ATMOS_GRIB_GRID_TMPL` directly in the yaml file where `'${MEMDIR}'` can be used for members as well? I tried something like this and it throws an undefined error.

```
        {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
                              '${RUN}':RUN,
                              '${YMD}':cycle_YMD,
                              '${HH}':cycle_HH,
                              '${MEMDIR}': 'ensstat' }) %}

        {% set COMIN_ATMOS_ENSSTAT_1p00 = ATMOS__GRIB_GRID_TMPL | replace_tmpl(tmpl_dict) %}
```